query/src/configs/config_storage.rs

```diff
@@ -0,0 +1,240 @@
+// Copyright 2020 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt;
+
+use structopt::StructOpt;
+use structopt_toml::StructOptToml;
+
+use crate::configs::Config;
+
+const DEFAULT_STORAGE_TYPE: &str = "DEFAULT_STORAGE_TYPE";
+
+// DFS Storage env.
+const DFS_STORAGE_ADDRESS: &str = "DFS_STORAGE_ADDRESS";
+const DFS_STORAGE_USERNAME: &str = "DFS_STORAGE_USERNAME";
+const DFS_STORAGE_PASSWORD: &str = "DFS_STORAGE_PASSWORD";
+const DFS_STORAGE_RPC_TLS_SERVER_ROOT_CA_CERT: &str = "DFS_STORAGE_RPC_TLS_SERVER_ROOT_CA_CERT";
+const DFS_STORAGE_RPC_TLS_SERVICE_DOMAIN_NAME: &str = "DFS_STORAGE_RPC_TLS_SERVICE_DOMAIN_NAME";
+
+// Disk Storage env.
+const DISK_STORAGE_DATA_PATH: &str = "DISK_STORAGE_DATA_PATH";
+
+// S3 Storage env.
+const S3_STORAGE_REGION: &str = "S3_STORAGE_REGION";
+const S3_STORAGE_KEY: &str = "S3_STORAGE_KEY";
```

Comment:
consider rename it to S3_STORAGE_ACCESS_KEY_ID to comply with official api definition in AWS or COS